ci-operator/step-registry/workers-rhel/pre-hook/fips/workers-rhel-pre-hook-fips-commands.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -x
+
+
+if [[ "${FIPS_ENABLED:-}" == "false" ]]; then
+  echo "Precondition not matched! Should be a fips enabled cluster!"
+  exit 1
+fi
+
+# Ensure our UID, which is randomly generated, is in /etc/passwd. This is required
+# to be able to SSH.
+if ! whoami &> /dev/null; then
+    if [[ -w /etc/passwd ]]; then
+        echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:${HOME}:/sbin/nologin" >> /etc/passwd
+    else
+        echo "/etc/passwd is not writeable, and user matching this uid is not found."
+        exit 1
+    fi
+fi
+
+platform=$(oc get infrastructure cluster -o=jsonpath='{.status.platform}')
+export PLATFORM=${platform}
```

Comment:
Suggest to merge the above 2 lines:
```
export PLATFORM=$(oc get infrastructure cluster -o=jsonpath='{.status.platform}')
```

Reply:
```
 export PLATFORM=$(oc get infrastructure cluster -o=jsonpath='{.status.platform}')
       ^------^ SC2155: Declare and assign separately to avoid masking return values. 
```
It will report [error](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/33009/pull-ci-openshift-release-master-step-registry-shellcheck/1582215580763557888) after merged. Will change it back.

Reply:
Because `platform` var name is also used in ansible playbook, for easy grep, it is better to remove it here, how about this:
```
export PLATFORM=""
PLATFORM=$(oc get infrastructure cluster -o=jsonpath='{.status.platform}')
```